docs/source/configuration/telemetry.md

```diff
@@ -31,38 +31,32 @@ which will prompt you for your consent the first time.
 For technical information on how the telemetry collection works, you can browse
 [the source code of `kedro-telemetry`](https://github.com/kedro-org/kedro-plugins/tree/main/kedro-telemetry).
 
-## How do I consent to the use of Kedro-Telemetry?
-
-Kedro-Telemetry is a Python plugin. To install it:
-
-```console
-pip install kedro-telemetry
-```
-
-```{note}
-If you are using an official [Kedro project template](/starters/starters) then `kedro-telemetry` is included in the [project-level `requirements.txt`](/kedro_project_setup/dependencies) of the starter. `kedro-telemetry` is activated after you have a created a new project with a [Kedro project template](/starters/starters) and have run `kedro install` from the terminal.
-```
+## How do I withdraw consent?
 
-When you next run the Kedro CLI you will be asked for consent to share usage analytics data for the purposes explained in the privacy notice, and a `.telemetry` YAML file will be created in the project root directory. The variable `consent` will be set according to your choice in the file, e.g. if you consent:
+To withdraw consent, you have a few options:
 
-```yaml
-consent: true
-```
+1. **Set Environment Variables**:
+   Set the environment variables `DO_NOT_TRACK` or `KEDRO_DISABLE_TELEMETRY` to any value. The presence of these environment variables will disable telemetry for all Kedro projects in that environment and will override any consent specified in the `.telemetry` file of the specific project.
 
-```{note}
-The `.telemetry` file should not be committed to `git` or packaged in deployment. In `kedro>=0.17.4` the file is git-ignored.
-```
+2. **CLI Option When Creating a New Project**:
+   When creating a new project, you can use the command:
 
-## How do I withdraw consent?
+   ```console
+   kedro new --telemetry=yes/no
+   ```
+   This will create a new project with a `.telemetry` file in its root folder, containing `consent: true/false` accordingly. This file will be used when executing Kedro commands within that project folder. Note that telemetry data about the execution of the `kedro new` command will still be sent if telemetry has not been disabled using environment variables.
```

Comment:
I agree that it can be simpler for us to only switch the state to `False`. So the telemetry is on by default and all the options allow to turn it off.

Reply:
In fact, today a user told me that they were confused by this option

Reply:
I agree with you, changed it